app/Config/DocTypes.php

```diff
@@ -30,4 +30,14 @@ class DocTypes
         'xhtml-rdfa-1'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">',
         'xhtml-rdfa-2'      => '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.1//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-2.dtd">',
     ];
+
+    /**
+     * Whether to add the solidus (`/`) character for void HTML elements (e.g. `<input>`)
```

Comment:
Sorry, my thoughts were mixed earlier.
```suggestion
     * Whether to remove the solidus (`/`) character for void HTML elements (e.g. `<input>`)
```